docs/manifests/README.md

```diff
@@ -4,6 +4,8 @@ Rather than using scripts AppGet uses `YAML` files to define application package
 
 All available packages are hosted in our official [GitHub Repository](https://github.com/appget/appget.packages). You can review and propose changes to any of the manifests using a pull-request.
 
+To create a new manifest and propose its addition to the repository, use the [`create` command](/commands/#create).
```

Comment:
This path structure should work on https://docs.appget.net/ but won't work correctly when browsing this repo locally. I'm open to other ideas for how to construct this path.

Reply:
If you leave off the leading slash, it's supposed to work as a relative path, I think?

Reply:
Leaving off the trailing slash is another option that works on docs.appget.net but not when browsing the repo locally, since `manifests` is a directory. There may not be a link scheme that works in both, but I was highlighting this because I don't know that the intended style in this repo is.